doc/classes/AudioEffectInstance.xml

```diff
@@ -13,11 +16,15 @@
 			<param index="1" name="dst_buffer" type="AudioFrame*" />
 			<param index="2" name="frame_count" type="int" />
 			<description>
+				Called by the [AudioServer] to process this effect. When [method _process_silence] is not overridden or it returns [code]false[/code], this method is called only when the bus is active.
+				[b]Note:[/b] It is not useful to override this method in GDScript or C#. Only GDExtension can take advantage of it.
```

Comment:
Now, the one thing that could be verified is this about GDExtension. I've already asked on RocketChat but I've been left hanging so far. And it better be true, otherwise this method is actually pointless and not working as intended.

Reply:
https://chat.godotengine.org/channel/audio?msg=KAsTbQsmDTyoGrh8n
> All right, looked it up, extensions can use this. There are GDVIRTUAL calls in there, and arguments come up as pointers in GodotCpp. The fact it appears also in the script API is an odd byproduct of that tho